specification/behaviors.md

```diff
@@ -46,3 +46,8 @@ Logs](https://github.com/open-telemetry/opentelemetry-specification/tree/main/sp
 The logs containing profiling data MUST be sent via OTLP. Instrumentation
 libraries SHOULD reuse persistent OTLP connections from other signals (traces,
 metrics).
+
+## Trace State Interchange
```

Comment:
We discussed this in person, but have you given any more thought to the idea of using one or more separate http headers rather than `tracestate`?  This would seem to fit the "one hop" requirement much better than abusing the "trace-wide" `tracestate`.  If not, then you need to spell out the "receive and delete" one-hop nature of these fields (without which the ability to link between the product sets gets horribly confused)

Reply:
Uh. I was attempting (probably poorly) to remain consistent with other design docs I started with. Those docs used the word `traceState` which I took to mean the actual `tracestate` spec/header, but I guess I'm mistaken....or more likely that's just a poor choice when some parties aren't doing tracing anyway.

Let's just cook up something else custom then.

Reply:
migrating to separate headers now.